managed/services/grafana/auth_server_test.go

```diff
@@ -167,7 +183,13 @@ func TestAuthServerAuthenticate(t *testing.T) {
 
 	ctx := context.Background()
 	c := NewClient("127.0.0.1:3000")
-	s := NewAuthServer(c, checker, nil)
+
+	roleRegistry := roles.NewRegistry(map[roles.EntityType]roles.EntityService{
+		roles.EntityUser: roles.NewUser("user_id", func() roles.EntityModel {
+			return &models.UserRoles{}
+		}, models.UserRolesView),
+	})
```

Comment:
we already have interfaces in this package, so we can mock this registry in this test.